From c192181111eaedb39747a8f5562eeffc7ba4ea90 Mon Sep 17 00:00:00 2001
From: Mark Rushakoff <mark@strange.love>
Date: Fri, 13 Dec 2024 11:09:26 -0500
Subject: [PATCH 6/6] chore: temporarily disable server v2 rest and grpc

The config values aren't getting wired through here properly, and at
this point I can't tell if it's a gcosmos wiring issue or a serverv2
bug, so just hardcode them to disabled for now.
---
 server/v2/api/grpc/server.go | 5 +++++
 server/v2/api/rest/server.go | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/server/v2/api/grpc/server.go b/server/v2/api/grpc/server.go
index 5775a6fd0e..bdb4c50374 100644
--- a/server/v2/api/grpc/server.go
+++ b/server/v2/api/grpc/server.go
@@ -227,6 +227,11 @@ func (s *Server[T]) Config() any {
 }
 
 func (s *Server[T]) Start(ctx context.Context) error {
+	const temporarilyDisable = true
+	if temporarilyDisable {
+		return nil
+	}
+
 	if !s.config.Enable {
 		s.logger.Info(fmt.Sprintf("%s server is disabled via config", s.Name()))
 		return nil
diff --git a/server/v2/api/rest/server.go b/server/v2/api/rest/server.go
index 1ea098c6dc..9356dde616 100644
--- a/server/v2/api/rest/server.go
+++ b/server/v2/api/rest/server.go
@@ -67,6 +67,11 @@ func (s *Server[T]) Name() string {
 }
 
 func (s *Server[T]) Start(ctx context.Context) error {
+	const temporarilyDisable = true
+	if temporarilyDisable {
+		return nil
+	}
+
 	if !s.config.Enable {
 		s.logger.Info(fmt.Sprintf("%s server is disabled via config", s.Name()))
 		return nil
-- 
2.39.5 (Apple Git-154)

